com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -1997,6 +1997,15 @@ internal void SceneChangedUpdate(Scene scene, bool notify = false)
         /// </remarks>
         private void Update()
         {
+            foreach (var behaviour in ChildNetworkBehaviours)
```

Comment:
We should discuss this as this could mean added performance hit for certain users based on how they use NetworkObject. It was my fault in adding the Update to NetworkObject in the first place, but perhaps we can figure out a way to hook the NetworkVariable update into the NetworkManager.NetworkUpdate method?